### ISSUE TITLE
a simple issus about slow optimization

### ISSUE THREAD

**Issue by Jackie-Chou** [lengstrom/fast-style-transfer#99]
After reading raw papers mentioned in readme and your codes, it seems to me that there's some small difference between your implementation and methods in those papers and it makes me confused, so I want to make sure about it.
As I understand it, loss network is just used to compute loss and its parameters should be fixed during optimizing and transferring both in fast and slow version, however, in your code optimize.py, the optimization is done using tf.train.AdamOptimizer(learning_rate).minimize(loss), which seems to optimize the loss w.r.t all variables in both graphs(transformation graph and loss graph), and I haven't found any code for fixing the parameters of loss network.
Maybe it's because of my unawareness, but I do hope you can help me clarify it.

**Comment by lengstrom** [comment]
In the slow implementation the gradient of the loss wrt the weights is always 0.
